### PR TITLE
HCIDOCS-194: SNO Worker Node Support Specification

### PR DESCRIPTION
**Pull request by kdrake20**
[HCIDOCS-194](https://issues.redhat.com//browse/HCIDOCS-194): SNO Worker Node Support Specification

Version(s):
4.14+

[Preview](https://73424--ocpdocs-pr.netlify.app/openshift-enterprise/latest/nodes/nodes/nodes-sno-worker-nodes) 

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by kdrake20** [comment]
**Who:** @omertuc, SME
**Contact Method:** Slack, Direct Message
**Timestamp:** Thursday, March 21, 2024 1:46 PM
**Summary:** Omer, can I please get an /lgtm for this content? Thanks!
**Next Steps:**
-[ ] Receive /lgtm from SME

**Comment by kdrake20** [comment]
**Who:** @trewest, QE 
**Timestamp:** Thursday, March 21, 2024 1:46 PM
**Summary:** Trey, can I please get an /lgtm for this content? Thanks!

**Next Steps:**
- [ ] Receive /lgtm from QE

**Review by trewest — APPROVED** [review]
/lgtm - looks fine to me as this is the number of workers we test in our CI but will defer to Omer regarding the performance findings

**Comment by kdrake20** [comment]
**Contact Method:** Slack, [#team-tps-docs](https://redhat-internal.slack.com/archives/C02GSQ4CSTE/p1711134875392749)

**Timestamp:** Friday, March 22, 2024 3:17 PM

**Summary:** Requested peer review.

**Next Steps:**

- [ ] Receive /lgtm from peer review.

**Comment by kquinn1204** [comment]
I rephrased second line to "Exceeding the recommended number of worker nodes may result in lower overall performance, and could even result in cluster failure. " I just felt  I wanted to draw clear distinction between lower performance and what would be a more severe failure.

File: nodes/nodes/nodes-sno-worker-nodes.adoc
Comment (apinnick) [inline nodes/nodes/nodes-sno-worker-nodes.adoc:20]:
```suggestion
Adding worker nodes does not expand the cluster control plane, and it does not provide high availability to your cluster. For {sno} clusters, high availability is handled by failing over to another site. When adding worker nodes to {sno} clusters, a tested maximum of two worker nodes are recommended. Exceeding the recommended number of worker nodes might result in lower overall performance, including cluster failure.
```
See ISG word usage for "may".

**Comment by kdrake20** [comment]
Contact Method: Slack, [#team-tps-docs](https://redhat-internal.slack.com/archives/C02GSQ4CSTE/p1711390382175289)

Timestamp: Monday, March 25, 2024 2:13 PM

Summary: Requested peer review.

Next Steps:

 -[ ]Receive /lgtm from peer review.

**Comment by apinnick** [comment]
Please do not merge. This PR should merge to main, not 4.14.

**Comment by apinnick** [comment]
> **Contact Method:** Slack, [#team-tps-docs](https://redhat-internal.slack.com/archives/C02GSQ4CSTE/p1711390382175289)
> 
> **Timestamp:** Monday, March 25, 2024 2:13 PM
> 
> **Summary:** Requested peer review.
> 
> **Next Steps:**
> 
>     * [ ]  Receive `/lgtm` from peer review.

Please follow the OCP doc manual for requesting peer review.

**Comment by kdrake20** [comment]
**Timestamp:** Friday, March 29, 2024 2:27 PM

**Summary:** Added `/label peer-review-needed` to PR.

**Next Steps:**

-[ ]Receive `/lgtm `from peer review.

**Comment by GroceryBoyJr** [comment]
Hey @kdrake20  I'll peer review, but can you rebase, please?

**Comment by GroceryBoyJr** [comment]
> Please do not merge. This PR should merge to main, not 4.14.

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM

**Comment by apinnick** [comment]
**The PR still points to 4.14.**

**Comment by apinnick** [comment]
Closing PR. 
@kdrake20 Please create a PR with these changes for main, with CP to 4.14.

**Pull request closed without merging**
